en/dynamic_data_in_templates/README.md

```diff
@@ -39,7 +39,7 @@ So now we want published blog posts sorted by `published_date`, right? We alread
 Post.objects.filter(published_date__lte=timezone.now()).order_by('published_date')
 ```
 
-Now we put this piece of code inside the `blog/views.py` file by adding it to the function `def post_list(request)`, but don't forget to first add `from django.utils import timezone`:
+Now we put this piece of code inside the `blog/views.py` file (open it in the code editor) by adding it to the function `def post_list(request)`, but don't forget to first add `from django.utils import timezone`:
```

Comment:
This could also be phrased as `... file using the code editor. Add it to ..` for a bit smoother sentence.

Reply:
OK, I'll revise. Thanks!

Reply:
Revised in next commit.